go.mod

```diff
@@ -0,0 +1,166 @@
+	github.com/openshift/cluster-api-provider-ovirt v0.1.1-0.20200120072345-c29232e3f4ed
+	github.com/openshift/library-go v0.0.0-20191205152556-73e1fb871a9b
+	github.com/openshift/machine-api-operator v0.2.1-0.20200116114648-a3345f5a437a
+	github.com/openshift/machine-config-operator v4.2.0-alpha.0.0.20190917115525-033375cbe820+incompatible
+	github.com/ovirt/go-ovirt v4.3.4+incompatible
+	github.com/ovirt/terraform-provider-ovirt v0.4.3-0.20200122105935-65b89ad00553
+	github.com/packer-community/winrmcp v0.0.0-20180921211025-c76d91c1e7db // indirect
+	github.com/pborman/uuid v1.2.0
+	github.com/pierrec/lz4 v2.3.0+incompatible // indirect
+	github.com/pkg/errors v0.9.1
+	github.com/pkg/sftp v1.10.1
+	github.com/prometheus/client_golang v1.2.1 // indirect
+	github.com/prometheus/client_model v0.0.0-20191202183732-d1d2010b5bee // indirect
+	github.com/prometheus/procfs v0.0.8 // indirect
+	github.com/satori/uuid v1.2.0 // indirect
+	github.com/shurcooL/httpfs v0.0.0-20190707220628-8d4bc4ba7749 // indirect
+	github.com/shurcooL/vfsgen v0.0.0-20181202132449-6a9ea43bcacd
+	github.com/sirupsen/logrus v1.4.2
+	github.com/spf13/cobra v0.0.5
+	github.com/stoewer/go-strcase v1.1.0 // indirect
+	github.com/stretchr/testify v1.4.0
+	github.com/terraform-providers/terraform-provider-aws v1.60.1-0.20200130210954-e03ec66d8774
+	github.com/terraform-providers/terraform-provider-azuread v0.7.0 // indirect
+	github.com/terraform-providers/terraform-provider-azurerm v1.41.0
+	github.com/terraform-providers/terraform-provider-google v1.20.1-0.20200204003432-77547e3e7d52
+	github.com/terraform-providers/terraform-provider-ignition v1.2.1
+	github.com/terraform-providers/terraform-provider-local v1.4.0
+	github.com/terraform-providers/terraform-provider-openstack v1.25.0
+	github.com/terraform-providers/terraform-provider-random v1.3.2-0.20190925210718-83518d96ae4f
+	github.com/terraform-providers/terraform-provider-vsphere v1.14.0
+	github.com/vincent-petithory/dataurl v0.0.0-20191104211930-d1553a71de50
+	github.com/vmware/govmomi v0.22.1
+	go.uber.org/atomic v1.5.1 // indirect
+	go.uber.org/multierr v1.4.0 // indirect
+	go.uber.org/zap v1.13.0 // indirect
+	golang.org/x/crypto v0.0.0-20191206172530-e9b2fee46413
+	golang.org/x/lint v0.0.0-20191125180803-fdd1cda4f05f
+	golang.org/x/oauth2 v0.0.0-20191202225959-858c2ad4c8b6
+	golang.org/x/sys v0.0.0-20191206220618-eeba5f6aabab
+	golang.org/x/time v0.0.0-20191024005414-555d28b269f0 // indirect
+	golang.org/x/xerrors v0.0.0-20191204190536-9bdfabe68543 // indirect
+	google.golang.org/api v0.14.0
+	gopkg.in/AlecAivazis/survey.v1 v1.8.7
+	gopkg.in/ini.v1 v1.51.0
+	gopkg.in/yaml.v2 v2.2.7
+	k8s.io/api v0.17.1
+	k8s.io/apimachinery v0.17.1
+	k8s.io/client-go v12.0.0+incompatible
+	k8s.io/klog v1.0.0
+	k8s.io/utils v0.0.0-20191114200735-6ca3b61696b6
+	sigs.k8s.io/cluster-api-provider-aws v0.4.7
+	sigs.k8s.io/cluster-api-provider-azure v0.2.2
+	sigs.k8s.io/cluster-api-provider-openstack v0.2.0
```

Comment:
nit: since we need to force using the fork, maybe we use `v0.0.0` for these?

Reply:
:+1: sometimes `go mod tidy` replaces them with a real version again, but I haven't quite figured out when exactly that happens. Will put a `v0.0.0` here once more!

I'll do the (hopefully) final rebase of this PR in a moment! (dinner first :)